gef-ui/server/generateApp.js

```diff
@@ -113,18 +114,10 @@ const generateApp = () => {
 
   app.use('/', routes);
 
-  // eslint-disable-next-line no-unused-vars
-  app.use((error, req, res, next) => {
-    if (error.code === 'EBADCSRFTOKEN') {
-      // handle CSRF token errors here
-      res.status(error.statusCode || 500);
-      res.redirect('/');
-    } else {
-      res.render('partials/problem-with-service.njk', { user: req.session.user, error });
-    }
-  });
+  app.get('*', (req, res) => res.status(404).render('partials/page-not-found.njk', { user: req.session.user }));
+
+  app.use(getFrontEndErrorHandler('partials/problem-with-service.njk'));
```

Comment:
I think we should have this before the `page-not-found` handler

Reply:
It should be last according to the [docs](https://expressjs.com/en/guide/error-handling.html) -- `You define error-handling middleware last, after other app.use() and routes calls`